.github/CODEOWNERS

```diff
@@ -41,6 +41,10 @@
 /addons/website*/ @odoo/rd-website
 /addons/website_event*/ @odoo/rd-notif-muted @odoo/rd-sm
 /addons/website_slides*/ @odoo/rd-notif-muted @odoo/rd-sm
+# Logistics
+/addons/*stock*/ @odoo/rd-logistics
+/addons/*mrp*/ @odoo/rd-logistics
+/addons/*purchase*/ @odoo/rd-logistics
```

Comment:
Hello,

I'm afraid those wildcard rules are getting out of hand.
They should be the exception not the norm due to the unfortunate "last matching rule take priority" that github enforces with this feature.

As it is, this will shadow a lot of existing rules, including those from `website*` for `website_sale_stock` and such, just by being "after" them in the file.

Due to this limitation and as this file grows I think we should stop being lazy with wildcards and actually list one by one all the modules that are concerned with specific rules.

How many of them would that be for your team? Do you have the full list available?

PS: do you actually intend to target master and not v12? This will prevent you from catching fixes in earlier version (forward-port will not trigger the rule).

Reply:
Hello,

Thank you for your help.

I understand your concerned :
- I made a list of module that we want to get notification (note some doesn't exist yet, but it should merge soon, it is a issue ?)
- I created a PR for the 12.0 (I am going to close this one) : https://github.com/odoo/odoo/pull/68795 (because we re-target some modules doesn't exist in 12.0 but only created in future version, I should update the list for each version or it is not a big deal ?)